client/src/legacy/ArchiveAdmin.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * File: ArchiveAdmin.js
+ */
+import $ from 'jquery';
+
+$.entwine('ss', ($) => {
+  $('.ArchiveAdmin__action--restore').entwine({
+    onmatch() {
+      $(this).attr('readonly', false);
+      $(this).attr('disabled', false);
+    }
+  });
+
+  $('.other-model-selector select').entwine({
```

Comment:
These should always be scoped to the section they're applying to (`.ArchiveAdmin` or something). It's legacy, but let's not make legacy worse ;)